iroh-docs/src/engine/live.rs

```diff
@@ -379,11 +370,6 @@ impl<B: iroh_blobs::store::Store> LiveActor<B> {
     async fn shutdown(&mut self) -> anyhow::Result<()> {
         // cancel all subscriptions
         self.subscribers.clear();
-        // shutdown gossip actor
-        self.gossip_actor_tx
-            .send(ToGossipActor::Shutdown)
-            .await
-            .ok();
```

Comment:
can't we wait for the gossip actor exit anymore?

Reply:
there's no gossip actor anymore in the live engine! the `GossipState` is not a loop of its own, it just contains state and abortable handles to the topic loop tasks.

the main `Gossip` is passed into the live engine, so should be shutdown outside it as well.

Reply:
Pushed a commit that properly joins the topic task loops in the sync engine to catch panics. Also added a TODO to catch panics when aborthing other joinsets in the sync engine too.